app/main/helpers/validation.py

```diff
@@ -62,9 +62,6 @@ def get_error_message(self, question_id, message_key):
         return default_messages.get(
             message_key, 'There was a problem with the answer to this question')
 
```

Comment:
It defined the expected override in a subclass though. Without it you get a less friendly 'attribute not found' error if you forget the override in a new subclass.